src/workerd/jsg/util.c++

```diff
@@ -178,45 +193,50 @@ TunneledErrorType tunneledErrorType(kj::StringPtr internalMessage) {
     }
   };
 
-  auto tryExtractError =
-      [isFromRemote](kj::StringPtr msg) -> kj::Maybe<TunneledErrorType> {
+  auto tryExtractError = [isFromRemote, &isDurableObjectBroken](kj::StringPtr msg)
```

Comment:
nit: Since we aren't modifying `isDurableObjectBroken` and it's a primitive, it feels like we should pass a value.

Reply:
Ahhhh, good point! And it should all inline better if the lambda only has arguments anyway.